### DISCUSSION TITLE
Tor（洋葱）浏览器的使用教程

### DISCUSSION THREAD

**Discussion by TrafficLight6** [share121/inter-knot#1784] [其他]
![tor](https://github.com/user-attachments/assets/505cf075-bff0-4874-a3cc-f2b339f00058)
# 0 前言  
tor是一个安全浏览器，其初衷是保护用户隐私，可以免受数据追踪。不要用于非法用途。请遵守所在地区的法律与法规！！！  

# 1 安装  
[官网下载页面](https://www.torproject.org/zh-CN/download/)  
~~如果无法访问我也没办法，但是我在学校都可以访问和下载~~

# 2使用  
安装完毕后，打开浏览器，会叫你链接网桥。
![image](https://github.com/user-attachments/assets/643a99e1-f028-49fa-baad-3cc9c0880de5)   
但是在PRC大概率会链接失败，如果成功就直接跳过则一步
如果链接失败，点击右上角三条杠   
![image](https://github.com/user-attachments/assets/997e3dfe-0bd5-499a-97c2-fb51c8049599)   
点开设置   
![image](https://github.com/user-attachments/assets/ed982159-e959-4bdb-85af-455c5f09211d)   
点开链接   
![image](https://github.com/user-attachments/assets/cc6e1d4f-a36a-4a62-af13-5917d95a5f91)   
在跟换网桥中点击选择内置网桥   
![image](https://github.com/user-attachments/assets/f7c1d966-c6fb-406b-a98d-e97784c90f4f)   
如图配置   
![image](https://github.com/user-attachments/assets/cc06cebb-9385-4eee-90b2-43e41296b846)  
点击连接   
等待  
链接成功后，会跳转到搜索引擎duckduckgo，这是一款搜索引擎  
# 3后记
1.如图，懂？（左边是tor，右边是edge）
![image](https://github.com/user-attachments/assets/2d724a6c-934b-4351-b4ff-32b552c58820)
2.不要点开duckduckgo中的onionze，产生任何法律责任包括刑事责任由使用者自己责  
![image](https://github.com/user-attachments/assets/4d63cc72-4f06-46f0-8051-947d10099dc8)
# 4参考
[https://www.bilibili.com/video/BV1Ri42197ig/?spm_id_from=333.999.0.0&vd_source=c5a9da41bff36395200ce94f69b0c9f6](https://www.bilibili.com/video/BV1Ri42197ig/?spm_id_from=333.999.0.0&vd_source=c5a9da41bff36395200ce94f69b0c9f6)

**Comment by aizai1173** [comment]
请问用洋葱随便进暗网的话，电脑会中毒吗？（因为有人说风险很大所以从来没试过）

**Reply by owofile** [reply]
一般不会，如果你比较谨慎可以把安全挡位拉满，这样只提供最基础的阅读，不会加载Cookies和JS
